packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1007,6 +1007,8 @@ export interface CallWithChatAdapterManagement {
     downloadAttachment: (options: {
         attachmentUrl: string;
     }) => Promise<AttachmentDownloadResult>;
+    // (undocumented)
```

Comment:
Should we add some documentation here?

Reply:
This just needed to be regenerated but the documentation is already in place